com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -366,7 +366,10 @@ int GetDestination()
             }
 
             // Depth of Field
-            if (m_DepthOfField.IsActive() && !isSceneViewCamera)
+            // Adreno 3xx SystemInfo.graphicsShaderLevel is 35, but instancing support is disabled due to buggy drivers.
+            // DOF shader uses #pragma target 3.5 which adds requirement for instancing support, thus marking the shader unsupported on those devices.
+            var dofMaterial = m_DepthOfField.mode.value == DepthOfFieldMode.Gaussian ? m_Materials.gaussianDepthOfField : m_Materials.bokehDepthOfField;
```

Comment:
I dont' understand how the material returns null for Adreno 3xx here.

Reply:
If the shader is not supported, it returns null. That's a result of an older identical issue fix https://github.com/Unity-Technologies/Graphics/commit/9157ee7b686785a2b2e0e1cd903ea681fe65b83f